import_export/formats/base_formats.py

```diff
@@ -99,6 +106,21 @@ def can_import(self):
     def can_export(self):
         return hasattr(self.get_format(), 'export_set')
 
+    def _escape_html(self, dataset):
+        for _ in dataset:
+            row = dataset.lpop()
+            row = [html.escape(str(cell)) for cell in row]
```

Comment:
Do you think it might be useful to have DEBUG level logs for before and after here, or is that going too far for enabling troubleshooting?

Reply:
I would say probably overkill.  I think if users have enabled the flags, then they will expect data to be changed on output.  It would also flood the logs for large datasets.  However it might be a good idea to log information somewhere just in case users are confused by the output if they have forgotten that they enabled the flags.

Reply:
@pokken-magic have added debug logging which should be called as a one-off when export is called.